ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -115,6 +115,9 @@ message GetTransactionsRequest {
   // Optional
   TraceContext trace_context = 1000;
 
+  // If set to True, the flat transactions stream events will contain exercised events matching given filters.
+  // Optional
+  bool include_non_consuming_exercise_events = 6;
```

Comment:
This is an entry point for the feature - a flag in the `GetTransactionRequest`. This flag is then passed all the way down to the database queries to configure proper filtering of `participant_events` rows.